internal/controller/config/config.go

```diff
@@ -116,6 +108,19 @@ func (cm *ConfigManager) LoadConfig(onErrorReloading func(error), configPath str
 	return nil
 }
 
+// LoadConfigFromBuffer is  Specifically for tests
```

Comment:
```suggestion
// LoadConfigFromBuffer is specifically for tests
```

Reply:
Is this function an alternative to testing `LoadConfig()`? It's a little odd to have a different function being tested than the real one. We won't catch if that changes. Is it necessary?

Reply:
I used it to loadconfig in gitcontroller_tests ,I will see if this can be removed and we can use existing LoadConfig